tests/lapi/bitop_arshift_test.lua

```diff
@@ -17,8 +17,10 @@ local arshift = bitop.arshift
 
 local function TestOneInput(buf)
     local fdp = luzer.FuzzedDataProvider(buf)
-    local x = test_lib.random_number(fdp)
-    local y = test_lib.random_number(fdp)
+    local MAX_INT = test_lib.MAX_INT()
```

Comment:
Maybe it is easier just to use `consume_integer()`? Here and below.

Reply:
Fixed.